jwst/coron/median_replace_img.py

```diff
@@ -101,3 +109,34 @@ def median_replace_img(img_model, box_size, bad_bitvalue):
         img_model.data[nimage] = img_int
 
     return img_model
+
+
+def separate_non_science_pixels(img_dq, bad_locations):
+    """
+    For the median filter, we don't care about the NON_SCIENCE pixels, but they
+    produce a ton of warnings that clog up the alignment algorithm and make it
+    run really slowly. In this function, we take all the bad pixels and pull out
+    the ones with NON_SCIENCE flags so we can set them to 0 without running the
+    median filter.
+
+    Parameters
+    ----------
+    img_dq: 2-D dq image
```

Comment:
Formatting of the parameter definitions here should follow the same syntax as what's seen above for `median_replace_img`

Reply:
Fixed.